### PR TITLE
okx - fetchTransfers

### PR DESCRIPTION
**Pull request by ttodua**
```
[
  {
    info: {
      bal: "123.876022204",
      balChg: "-13.761630400",
      billId: "552957285564453689",
      ccy: "USDT",
      execType: "",
      fee: "",
      from: "18",
      instId: "",
      instType: "",
      mgnMode: "",
      notes: "To Funding Account",
      ordId: "",
      pnl: "",
      posBal: "",
      posBalChg: "",
      price: "0.0",
      subType: "12",
      sz: "-13.7616304",
      to: "6",
      ts: "1678107293328",
      type: "1",
    },
    id: "552957285564453689",
    timestamp: 1678107293328,
    datetime: "2023-03-06T12:54:53.328Z",
    currency: "USDT",
    amount: 13.7616304,
    fromAccount: "trading",
    toAccount: "funding",
    status: undefined,
  },
]
```

### CODE REVIEW

File: ts/src/okx.ts [hunk 1]
```diff
@@ -759,7 +760,7 @@ export default class okx extends Exchange {
                 'fetchPositions': {
                     'method': 'privateGetAccountPositions', // privateGetAccountPositions or privateGetAccountPositionsHistory
                 },
-                'createOrder': 'privatePostTradeBatchOrders', // or 'privatePostTradeOrder' or 'privatePostTradeOrderAlgo'
+                'createOrder': 'privatePostTradeBatchOrders', // privateGetAccountBills or 'privatePostTradeOrder' or 'privatePostTradeOrderAlgo'
```
Comment (carlosmiei) [inline ts/src/okx.ts:763]: @ttodua how is this method related to `createOrder`?

Comment (ttodua) [reply ts/src/okx.ts:763]: uhoh, i misplaced that comment when pasting. fixed now

File: ts/src/okx.ts [hunk 2]
```diff
@@ -4855,17 +4808,47 @@ export default class okx extends Exchange {
...
-        const amount = this.safeNumber (transfer, 'amt');
+        let amount = this.safeNumber (transfer, 'amt');
         const fromAccountId = this.safeString (transfer, 'from');
         const toAccountId = this.safeString (transfer, 'to');
         const accountsById = this.safeValue (this.options, 'accountsById', {});
         const fromAccount = this.safeString (accountsById, fromAccountId);
         const toAccount = this.safeString (accountsById, toAccountId);
-        const timestamp = this.milliseconds ();
+        const timestamp = this.safeInteger (transfer, 'ts', this.milliseconds ());
         const status = this.safeString (transfer, 'state');
```
Comment (carlosmiei) [inline ts/src/okx.ts:4847]: @ttodua we're missing the status parsing here, example from  a different exchange:
```Javascript
    parseTransferStatus (status) {
        const statuses = {
            'prepared': 'pending',
            'confirmed': 'ok',
            'cancelled': 'cancelled',
            'waiting_for_admin': 'pending',
        };
        return this.safeString (statuses, status, status);
    }
```

Comment (ttodua) [reply ts/src/okx.ts:4847]: but the status-related info is not provided in the raw response. should we set it to "ok" directly?

Comment (carlosmiei) [reply ts/src/okx.ts:4847]: @ttodua in that case why do we have this line here: ` const status = this.safeString (transfer, 'state');`

Comment (ttodua) [reply ts/src/okx.ts:4847]: but that line might be for other methods (i.e. `fetchTransfer` as the commented response sample where `state` key appears, belongs to it, which was there before I added `fetchTransfers`).

Comment (carlosmiei) [reply ts/src/okx.ts:4847]: > but that line might be for other methods
@ttodua exactly, that is an issue because we're using the provided raw value instead of the unified one 😅

Comment (carlosmiei) [reply ts/src/okx.ts]: @ttodua thanks!

**Comment by carlosmiei** [comment]
LGTM ✅ 
```
p okx fetchTransfers "USDT" None 1
Python v3.10.9
CCXT v3.1.36
okx.fetchTransfers(USDT,None,1)
[{'amount': 1.0,
  'currency': 'USDT',
  'datetime': '2023-03-18T11:00:33.646Z',
  'fromAccount': 'trading',
  'id': '557277180943167489',
  'info': {'bal': '30.8172709885692353',
           'balChg': '-1.0000000000000000',
           'billId': '557277180943167489',
           'ccy': 'USDT',
           'execType': '',
           'fee': '',
           'from': '18',
           'instId': '',
           'instType': '',
           'mgnMode': '',
           'notes': 'To Funding Account',
           'ordId': '',
           'pnl': '',
           'posBal': '',
           'posBalChg': '',
           'price': '0',
           'subType': '12',
           'sz': '-1',
           'to': '6',
           'ts': '1679137233646',
           'type': '1'},
  'status': None,
  'timestamp': 1679137233646,
  'toAccount': 'funding'}]
```

**Review by carlosmiei — APPROVED** [review]
